lib/db/models/Swap.ts

```diff
@@ -1,34 +1,43 @@
-import Sequelize from 'sequelize';
-import * as db from '../../consts/Database';
+import { Sequelize, Model, DataTypes } from 'sequelize';
+import Pair from './Pair';
 
-export default (sequelize: Sequelize.Sequelize, dataTypes: Sequelize.DataTypes) => {
-  const attributes: db.SequelizeAttributes<db.SwapAttributes> = {
-    id: { type: dataTypes.STRING, primaryKey: true, allowNull: false },
-    fee: { type: dataTypes.INTEGER, allowNull: false },
-    pair: { type: dataTypes.STRING, allowNull: false },
-    orderSide: { type: dataTypes.INTEGER, allowNull: false },
-    status: { type: dataTypes.STRING, allowNull: true },
-    invoice: { type: dataTypes.STRING, unique: true, allowNull: false },
-    lockupAddress: { type: dataTypes.STRING, allowNull: false },
-  };
+class Swap extends Model {
+  public id!: string;
+  public fee!: number;
+  public pair!: string;
+  public orderSide!: number;
+  public status!: string;
+  public invoice!: string;
+  public lockupAddress!: string;
+  public createdAt!: string;
+  public updatedAt!: string;
 
-  const options: Sequelize.DefineOptions<db.SwapInstance> = {
-    tableName: 'swaps',
-    indexes: [
-      {
-        unique: true,
-        fields: ['id', 'invoice'],
-      },
-    ],
-  };
-
-  const Swap = sequelize.define<db.SwapInstance, db.SwapAttributes>('Swap', attributes, options);
+  public static load = (sequelize: Sequelize) => {
+    Swap.init({
+      id: { type: DataTypes.STRING, primaryKey: true, allowNull: false },
+      fee: { type: DataTypes.INTEGER, allowNull: false },
+      pair: { type: DataTypes.STRING, allowNull: false },
+      orderSide: { type: DataTypes.INTEGER, allowNull: false },
+      status: { type: DataTypes.STRING, allowNull: true },
+      invoice: { type: DataTypes.STRING, unique: true, allowNull: false },
+      lockupAddress: { type: DataTypes.STRING, allowNull: false },
+      createdAt: { type: DataTypes.STRING, allowNull: false },
+      updatedAt: { type: DataTypes.STRING, allowNull: true },
```

Comment:
Aren't `createdAt` and `updatedAt` created automatically and don't need to be initialized here?

Reply:
If I don't create them here I get an error when trying to access `createdAt` of an instance

`Property 'createdAt' does not exist on type 'Swap | ReverseSwap'.
  Property 'createdAt' does not exist on type 'Swap'`